pkgs/desktops/cursor/fuchsia-pop/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl }:
+
+
+stdenv.mkDerivation rec {
+  pname = "fuchsia-pop";
+  version = "2.0.0";
+
+  src =  fetchurl {
+    url = "https://github.com/ful1e5/fuchsia-cursor/releases/download/v2.0.0/Fuchsia-Pop.tar.gz";
+    sha256 = "b4dac495290146b0d9ed8cc782de1915c11b1887645358f12c196b03870f6265";
+  };
+
+  dontBuild = true;
+  dontConfigure = true;
+  noDumpEnvVars = true;
+
+  sourceRoot = ".";
+
+  installPhase = ''
+  runHook preInstall
+  mkdir -p $out/share/icons
+  cp -r ./Fuchsia-Pop* $out/share/icons
+  runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Fuchsia cursor Theme";
+    homepage = "https://github.com/ful1e5/fuchsia-cursor";
+    license = licenses.gpl3;
+    platforms = platforms.all;
+    maintainers = [ mataintainers.mrtuxa ];
+  };
+}
```

```suggestion
}

```
No newline at end of file